src/components/Avatar/types/main.ts

```diff
@@ -0,0 +1,26 @@
+import type React from 'react';
+
+import type {DistributiveOmit} from '../../../types/utils';
+import type {QAProps} from '../../types';
+import type {AvatarIconProps} from '../AvatarIcon';
+import type {AvatarImageProps} from '../AvatarImage';
+import type {AvatarTextProps} from '../AvatarText';
+
+import type {AvatarCommonProps, AvatarSize} from './common';
+
+export type AvatarTheme = 'normal' | 'brand';
+export type AvatarView = 'filled' | 'outlined';
+
+interface AvatarBaseProps extends QAProps {
+    size?: AvatarSize;
+    theme?: AvatarTheme;
+    view?: AvatarView;
+    backgroundColor?: string;
+    borderColor?: string;
+    title?: string;
+    ariaAttributes?: React.AriaAttributes;
+    className?: string;
```

Comment:
Duplicate prop from `AvatarCommonProps`

Reply:
This interface does not extended by `AvatarCommonProps`